samples/hello++/main.cpp

```diff
@@ -0,0 +1,40 @@
+#include <xboxrt/debug.h>
+#include <pbkit/pbkit.h>
+#include <hal/xbox.h>
+#include "stdio.h"
+
+class hello{
+private:
+  int hiddenValue = 0;
+public:
+  hello() {hiddenValue = 5;};
+  hello(int v) : hiddenValue(v) {};
+  int getValue() { return hiddenValue; };
+};
+
+int main() {
+  switch(pb_init())
+  {
+  case 0: break;
+  default:
+    XSleep(2000);
+    XReboot();
+    return -1;
+  }
+
+  pb_show_debug_screen();
+
+  hello h(6);
+  hello i;
```

Comment:
I suggest calling them `defaultFive` and `customSix` or something. The classname could be `NumberClass`